app/src/main/java/org/greenstand/android/TreeTracker/signup/CredentialEntryView.kt

```diff
@@ -109,7 +110,11 @@ fun CredentialEntryView(viewModel: SignupViewModel, state: SignUpState) {
                     ),
                     keyboardActions = KeyboardActions(
                         onGo = {
-                            viewModel.goToNameEntry()
+                            if (state.isEmailValid || state.isPhoneValid) {
+                                viewModel.goToNameEntry()
+                            } else {
+                                CustomToast.showToast("Please enter a valid email")
```

Comment:
Please use string resources for any text.

Reply:
@Elforama thanks! can you comment at all on which is the preferred way to go about things (i.e. making a snackbar or leaving a todo) regarding @EngrTaofeek 's suggestion above?

Reply:
Let's use the snackbar.